TestWeatherReport.cpp

```diff
@@ -0,0 +1,62 @@
+#include "gtest/gtest.h"
+#include "WeatherReport.h"
+#include "SensorStub.h"
+
+const int HUMIDITY_RAINY = 72;
+const int PRECIPITATION_RAINY = 70;
+const int TEMP_RAINY = 26;
+const int WINDSPEED_RAINY = 52;
+
+const int HUMIDITY_STORMY = 72;
+const int PRECIPITATION_STORMY = 70;
+const int TEMP_STORMY = 26;
+const int WINDSPEED_STORMY = 52;
+
+const int HUMIDITY_SUNNY = 72;

```

Comment:
All tests deal with a single value of humidity and windspeed. If we need to capture the behavior of the program in these tests, how would you rewrite them? Browse "property-based tests"